dts/arm/st/h7/stm32h723.dtsi

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2020 Alexander Kozhinov
+ * AlexanderKozhinov@yandex.com
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/h7/stm32h7.dtsi>
+
+/ {
+	/* DTCM memory directly coppled to CPU */
+	dtcm: memory@20000000 {
+		compatible = "arm,dtcm";
+		reg = <0x20000000 DT_SIZE_K(128)>;
+	};
+
+	/* AXI SRAM in D1 domain (AXI bus) */
+	sram0: memory@24000000 {
+		reg = <0x24000000 DT_SIZE_K(128)>;
+		compatible = "mmio-sram";
+	};
+
+	/* RAM shared between ITCM and AXI in D1 (AXI bus) */
+	sram1: memory@24020000 {
+		reg = <0x24020000 DT_SIZE_K(192)>;
+		compatible = "mmio-sram";
```

Comment:
SRAM1 is set to `0x30000000`

![image](https://user-images.githubusercontent.com/25011557/99254188-3e057700-2812-11eb-82cc-d7c93c873fd7.png)

This memory is actually the `itcm` (instruction tightly coupled RAM). It probably needs to have its own compatible, similar to `dtcm`. @erwango any comments?

Reply:
SRAM names were to more meaningful names similar to this from RM now.

Reply:
@erwango ping

Reply:
Changed to sram1